docs/data/sdks/typescript-browser/migration.md

```diff
@@ -0,0 +1,497 @@
+---
+title: Migration Guide
+description: This is a guide on migrating from amplitude-js
```

Comment:
this is the page's meta text, so can you add a little more detail here? for example: 

Use this guide to easily migrate from Amplitude's legacy browser SDK (amplitude-js) to the new SDK (@amplitude/analytics-browser).

Reply:
```suggestion
description: Use this guide to easily migrate from Amplitude's legacy browser SDK (amplitude-js) to the new SDK (@amplitude/analytics-browser).
```